server/graphql/v1/types.js

```diff
@@ -869,6 +869,72 @@ export const CommentType = new GraphQLObjectType({
   },
 });
 
+export const NotificationType = new GraphQLObjectType({
+  name: 'NotificationType',
+  description: 'This represents a Notification',
+  fields: () => {
+    return {
+      id: {
+        type: GraphQLInt,
+        resolve(notification) {
```

Comment:
Not a requested change, just an info: when the field map the schema it is not necessary to add a `resolve` function. GraphQL resolves to `model => model[fieldName]` by default.

Reply:
Oh, just knowing this. Thanks.